app/src/main/java/com/babylon/wallet/android/presentation/wallet/AccountCardView.kt

```diff
@@ -194,18 +195,18 @@ fun AccountCardPreview() {
     RadixWalletTheme {
         Column(verticalArrangement = Arrangement.spacedBy(16.dp)) {
             AccountCardView(
-                accountWithResources = AccountWithResources(
+                accountWithAssets = AccountWithAssets(
                     account = SampleDataProvider().sampleAccount(),
-                    resources = Resources(
-                        fungibleResources = listOf(
+                    assets = Assets(
+                        fungibles = listOf(
                             Resource.FungibleResource(
-                                resourceAddress = "resource_address",
+                                resourceAddress = XrdResource.address(),
```

Comment:
I think you should rollback any changes to previews and use static data, becuase if you use RET calls (which `address()`), it breaks every preview when RET is referenced implicitly.